src/screens/List.js

```diff
@@ -484,6 +484,10 @@ const ListPage = () => (
           <Description>Any additional style for the list items.</Description>
           <GenericExtend />
         </Property>
+        <Property name="list.primaryKey">
+          <Description>Primary key styling accepts any Text props.</Description>
+          <Example>{`{{ color: ..., weight: ... }}`}</Example>
```

Comment:
Can we add a `PropertyValue`?
```suggestion
           <PropertyValue type="object">
             <Example>{`{{ color: ..., weight: ... }}`}</Example>
          </PropertyValue>
```

Reply:
yes I saw to weight is bold for our base theme so Ill add that